server/routes/auth_routes.py

```
@@ -140,6 +140,12 @@ def register():
                 
                 logging.info(f"New user registered: {email[:3]}***@*** (role=admin, org={org_id})")
 
+                try:
+                    from utils.auth.command_policy import seed_default_command_policy
+                    seed_default_command_policy(org_id, user_id)
+                except Exception as policy_err:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)